### PR TITLE
Handles the backend changes to the history API

### LINKED ISSUE #29: Accommodate history API changes
The history API has recently had breaking changes, which the SDK will need to accommodate. Since there are more changes expected soon (with respect to cursor handling), we will ideally only have breaking changes in the SDK once.

While history is not yet officially supported in the SDK, we may want to consider deprecating older versions (with https://docs.npmjs.com/cli/deprecate).

This should probably be treated as a breaking change that warrants a minor version change to v0.4 as the fetchHistory signature will be changing. This is slightly debatable, since again, fetchHistory isn't officially supported yet.

### PR DESCRIPTION
**Pull request by iredelmeier**
Closes #29

### CODE REVIEW

**Comment by iredelmeier** [comment]
We should hold off on merging this until we confirm that it will handle the upcoming cursor changes.

File: flux-sdk-browser/example/example.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-var CLIENT_ID = 'replace-with-your-client-id';
+var CLIENT_ID = '9de915f7-8808-4d2a-a8e3-7dda1f2c0a58';
```
Comment (daishiflux) [inline flux-sdk-browser/example/example.js]: Use test client id if any?

File: flux-sdk-common/spec/unit/serializers/history-serializer-spec.js [hunk 2]
```diff
@@ -29,21 +29,21 @@ describe('serializers.historySerializer', function() {
         this.serializedHistory = serialize(historyResponse);
       });
 
-      it('should set page and limit as 0', function() {
-        expect(this.serializedHistory.page).toEqual(0);
+      it('should set page and limit accordingly', function() {
```
Comment (daishiflux) [inline flux-sdk-common/spec/unit/serializers/history-serializer-spec.js]: Nit: Replace "page" with "cursor" in test description.

File: flux-sdk-common/spec/unit/serializers/history-serializer-spec.js [hunk 3]
```diff
@@ -29,21 +29,21 @@ describe('serializers.historySerializer', function() {
...
-        const historyResponse = historyFactory({ page: 2, limit: 5 });
+        const historyResponse = historyFactory({ limit: 5, cursor: '3' });
         this.serializedHistory = serialize(historyResponse);
       });
 
-      it('should set page and limit', function() {
-        expect(this.serializedHistory.page).toEqual(2);
+      it('should set the page and cursor limit', function() {
         expect(this.serializedHistory.limit).toEqual(5);
+        expect(this.serializedHistory.cursor).toEqual('3');
```
Comment (daishiflux) [inline flux-sdk-common/spec/unit/serializers/history-serializer-spec.js]: Please don't embed verification of the cursor here. The cursor should be treated as an opaque blob which is only valid to specify the continuation of the "running" query, and we don't want to commit the service to supporting any specific cursor format/representation.

File: flux-sdk-common/spec/unit/models/data-table-spec.js [hunk 4]
```diff
@@ -165,7 +165,7 @@ describe('models.DataTable', function() {
     it('should be able to add additional options', function(done) {
       this.dataTable.fetchHistory({
         cellIds: ['CELL1', 'CELL2'],
-        page: 2,
+        cursor: '2',
```
Comment (daishiflux) [inline flux-sdk-common/spec/unit/models/data-table-spec.js]: Elaborated on in a separate comment below, but please don't embed literal cursors in tests.

**Comment by iredelmeier** [comment]
This fails when tested against prod because the back end changes haven't deployed yet.

**Pull request merged**